presto-jdbc.rb

```diff
@@ -1,9 +1,9 @@
 class PrestoJdbc < Formula
   desc "JAR file for connecting to Presto over JDBC"
-  homepage "https://prestodb.github.io/docs/0.223/installation/jdbc.html"
-  url "https://repo1.maven.org/maven2/com/facebook/presto/presto-jdbc/0.223/presto-jdbc-0.223.jar"
-  sha256 "44ba89800257b085e8e0fc7dd95e58f9f767fa69f543f693b6f6f58937181aa5"
-  version "0.223"
+  homepage "https://prestosql.io/docs/current/installation/jdbc.html"
```

Comment:
I would hard lock this to 338 or we will have the newest version constantly being displayed

Reply:
I couldn't find a way to do this with the new docs, but i'll try again